NicolaeChristof/Descend-Into-Darkness
# Feature Lighting + Basic Camp

This PR is mostly focused on lighting. I've attached a torch to the character that lets them see things in their immediate vicinity. I've also included some of the glowing mushrooms that we've talked about. The scene is generally darker, but it does have some ambient lighting from the skylight (whose intensity can be raised or lowered).

Additionally, I've added the ability to place the camp down. If you hit the P button when near a pole (there's only one currently in the stage. It's to the right of the entrance to the cave, on the surface) it'll spawn a camp (which provides a bit of lighting as well).

How to test:
Make sure that the scene isn't too dark and that the scene is bright enough that you're able to make it to the goal. Additionally, make sure that the camp spawns when you hit the P near a pole. It doesn't currently let you respawn at it as I haven't merged in the health system yet.

Known bugs:
If you're right near the pole when you hit P, the tent will spawn around you.
Also, there seem to be 2 player blueprints (one in the SideScrollerCPP/Blueprints folder and one in the Assets/Blueprints folder). I'm assuming we're using the one in the assets folder, but I left the one in the SideScrollerCPP folder to make sure there aren't any compatibility issues.
